### PR TITLE
update manifest to support mlperf-logging for maxtext

### PR DESCRIPTION
**Pull request by kocchop**
Added the mlperf-logging to the manifest.yaml and also verified it using this [run](https://github.com/NVIDIA/JAX-Toolbox/actions/runs/7907539388)

### CODE REVIEW

File: .github/container/manifest.yaml [hunk 1]
```diff
@@ -148,3 +148,8 @@ language-to-reward-2023:
   tracking_ref: main
   latest_verified_commit: abb8e5125e4ecd0da378490b73448c05a694def5
   mode: git-clone
+mlperf-logging:
+  url: https://github.com/mlcommons/logging.git
+  tracking_ref: main 
+  latest_verified_commit: 38709131757a786d7f66150243b49eb0365324d7
+  mode: git-clone
```
Comment (terrykong) [inline .github/container/manifest.yaml:155]: You can set it to `pip-vcs` to document that this isn't source that we clone, but used just during pip-compile + pip-sync
```suggestion
  mode: pip-vcs
```

**Review by terrykong — CHANGES_REQUESTED** [review]
Review was a standalone comment

File: .github/container/manifest.yaml (hunk 1 above)
Comment (kocchop) [reply .github/container/manifest.yaml]: changed it to pip-vcs

**Review by terrykong — APPROVED** [review]
Lgtm

**Pull request merged**